cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -37,6 +37,19 @@ import (
 	kubeschedulerconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"
 )
 
+var defaultTopologySpreadConstraints = []corev1.TopologySpreadConstraint{
+	{
+		MaxSkew:           1,
+		TopologyKey:       corev1.LabelHostname,
+		WhenUnsatisfiable: corev1.ScheduleAnyway,
+	},
+	{
+		MaxSkew:           1,
+		TopologyKey:       corev1.LabelZoneFailureDomain,
```

Comment:
this is a non-GA label that is being promoted to GA [now](https://github.com/kubernetes/kubernetes/pull/81431). are we adding new uses of this label in this PR?

Reply:
Updated.